script/bootstrap-redhat

```diff
@@ -21,6 +21,8 @@ fi
 
 #yum update -y
 
+yum groupinstall -y "Development Tools"
```

Comment:
Do we need this for RHEL 6 as well? If not, might be better to put in the RHEL 7 logic at https://stackstorm.slack.com/archives/stackstorm/p1444154729001007